openverse_catalog/dags/util/loader/test_sql.py

```diff
@@ -554,27 +585,61 @@ def test_upsert_records_replaces_data(postgres_with_load_and_image_table, tmpdir
     TITLE_B = "Bobs Great Pic"
     META_DATA_B = '{"description": "Bobs cool picture"}'
 
-    load_data_query_a = (
-        f"INSERT INTO {load_table} VALUES("
-        f"'{FID}','{LAND_URL_A}','{IMG_URL_A}','{THM_URL_A}',"
-        f"'{WIDTH_A}','{HEIGHT_A}','{FILESIZE}','{LICENSE_A}','{VERSION_A}',"
-        f"'{CREATOR_A}','{CREATOR_URL_A}','{TITLE_A}','{META_DATA_A}',"
-        f"'{TAGS}','{WATERMARKED}','{PROVIDER}','{SOURCE}'"
-        f");"
+    query_values = create_query_values(
+        {
+            col.FOREIGN_ID: FID,
+            col.LANDING_URL: LAND_URL_A,
+            col.DIRECT_URL: IMG_URL_A,
+            col.THUMBNAIL: THM_URL_A,
+            col.FILESIZE: FILESIZE,
+            col.LICENSE: LICENSE_A,
+            col.LICENSE_VERSION: VERSION_A,
+            col.CREATOR: CREATOR_A,
+            col.CREATOR_URL: CREATOR_URL_A,
+            col.TITLE: TITLE_A,
+            col.META_DATA: META_DATA_A,
+            col.TAGS: TAGS,
+            col.WATERMARKED: WATERMARKED,
+            col.PROVIDER: PROVIDER,
+            col.SOURCE: SOURCE,
+            col.INGESTION_TYPE: INGESTION_TYPE,
+            col.WIDTH: WIDTH_A,
+            col.HEIGHT: HEIGHT_A,
+        }
     )
+    load_data_query_a = f"""INSERT INTO {load_table} VALUES(
+       {query_values}
+       );"""
```

Comment:
Would it be useful to have a `make_insert_query` function for creating statements like this? I'm seeing a lot of similar definitions in this file.

Reply:
Can we create a new issue for this?..

Reply:
Definitely!